src/abyssal_tome/app.py

```
@@ -179,10 +188,13 @@ def append_span(spans_list: list[ft.TextSpan], text_content: str, style: ft.Text
     if text_content:
         spans_list.append(ft.TextSpan(text=text_content, style=style or ft.TextStyle(), on_click=on_click_handler))
 
+
 async def replace_special_tags(page: ft.Page, text_input: str) -> list[ft.TextSpan]:
     """
      """
      Parses input text for special tags, markdown styles, and links, converting them into styled TextSpan objects for display.
```

Ruff (0.11.9)

SyntaxError: Unexpected indentation

---

Indentation is not a multiple of 4

(E111)